x-pack/plugins/security_solution/public/common/components/links/index.tsx

```diff
@@ -61,6 +62,45 @@ export const PortContainer = styled.div`
 `;
 
 // Internal Links
+const UebaDetailsLinkComponent: React.FC<{
```

Comment:
there are a bunch of these link components that should be refactored because they share so much code, they are nearly identical except a couple arguments. didn't have time to do that here